src/module.ts

```diff
@@ -132,9 +132,9 @@ const loadLocales = async (moduleOptions: ModuleOptions) => {
   moduleOptions.locales = []
 
   for (const locale of locales) {
-    const text = await import(
-      await resolvePath(resolve(runtimeDir, 'locale', locale))
-    ).then((r: any) => r.default || r)
+    const rpath = await resolvePath(resolve(runtimeDir, "locale", locale))
+    const path = !rpath.toLowerCase().startsWith("file://") || !rpath.toLowerCase().startsWith("/") ? `file://${rpath}` : rpath
```

Comment:
I'm curious, how did you come up with those changes, especially the `file://` prefix?